treeherder/etl/bugzilla.py

```diff
@@ -36,10 +36,7 @@ def run(self):
         offset = 0
         limit = 500
 
-        # fetch new pages no more than 30 times
-        # this is a safe guard to not generate an infinite loop
-        # in case something went wrong
-        for i in range(1, 30 + 1):
+        while True:
```

Comment:
Any other larger value (eg 100 pages) just wouldn't make sense, since we'd either hit it again (in which case facepalm) or else never hit it, because we'll hit the 600s timeout first.

The only other way is if we used this pattern:
```python
foo()
while len(temp_bug_list) < limit: